wondrous-app/components/Comment/styles.tsx

```diff
@@ -118,10 +119,35 @@ export const EditorContainer = styled.div`
 export const EditorPlaceholder = styled.div`
   min-height: ${editorMinHeight - editorPadding * 2}px;
 `;
-export const AddCommentButton = styled(CreateFormPreviewButton)`
+export const AddCommentButton = styled(Button)`
```

Comment:
can we use button from `components/Button`? (you can check storyook by running `yarn storybook`)

Reply:
Updated